content/applications/hr/employees.rst

```diff
@@ -0,0 +1,13 @@
+:nosearch:
+:show-content:
+:hide-page-toc:
+:show-toc:
```

Comment:
If you move your content to this page, no need for any of these options now.
```suggestion
```

However, if you add sub-pages, you'll need this:
```suggestion
:show-content:
:show-toc:
```